scrapinghub/client/utils.py

```diff
@@ -106,6 +107,20 @@ def __init__(self, cls, client, key):
             self._proxy_methods(methods)
             self._wrap_iter_methods([method[0] for method in methods])
 
+        if issubclass(cls, MappingResourceType):
+            methods = ['ignore_fields', 'expire', 'save', 'liveget',
+                       ('iter', '__iter__')]
+            self._proxy_methods(methods)
+            # the following methods are special and should be set to class
```

Comment:
Are you sure it's a good idea to mix this into the _Proxy? If we need a proxy for mapping resources - why cannot we create a separate class for that? + I'm not sure it's a good idea to follow the same conventions for dict-like objects we do for resources (see https://github.com/scrapinghub/python-scrapinghub/pull/53/files#r104448204)